src/config.py

```diff
@@ -9,17 +6,23 @@ class Config:
   Class for loading and accessing the config file.
   """
 
-  def __init__(self):
-    self._json = {}
+  def __init__(self, configs: list[dict[str, str | bool]]):
+    self._config = {}
 
-  def load(self, config_filepath: str):
-    if not os.path.exists(config_filepath):
-      raise FileNotFoundError(f"{config_filepath} does not exist.")
+    for config in configs:
+      self._config |= config
 
-    with open(config_filepath, "r", encoding="utf-8") as f:
-      self._json = json.loads(f.read())
+    if not self._config:
+      raise FileNotFoundError("Configuration not found.")
 
-    return self
+  @staticmethod
+  def boolify_string_option(value: str):
```

Comment:
Need this function as if you just blindly cast "false" as as bool it will be True.  Needed more complexity than just a basic if statement in variable assignment as we want None to be a valid value as to overwrite a False/True from a lower priority entry